src/groups/mqb/mqbstat/mqbstat_clusterstats.h

```diff
@@ -64,8 +64,8 @@ class ClusterStats {
     struct PartitionEventType {
         // TYPES
         enum Enum {
+            /// Time in nanoseconds it took for the rollover operation.
```

Comment:
I find this, in combination with the `struct`'s comment above, baffling.  

src/groups/mqb/mqbs/mqbs_filestore.cpp:2914 looks like it's the only use, which makes it a little more clear what's going on, but yikes